static/js/marathon.js

```diff
@@ -57,6 +58,12 @@ let app = new Vue({
         saved: false,
 
         renderer: null,
+        hover: false,
+        loading: false,
+        blocker: false,
```

Comment:
what does this do?

Reply:
These are all states for previews

1. hover: determines whether or not to show the preview
2. loading: ensures that previews don't get stuck by handling edge cases like when the cursor is moving too fast
3. blocker: indicates whether the preview is for timebox or embedded hyperlinks since previews are triggered on hover for embedded hyperlinks and on click for the timebox. This creates a clickable overlay that allows a player to close the preview box by clicking anywhere outside it as opposed to having a dedicated close button